jsx/Form.js

```diff
@@ -1365,6 +1365,48 @@ StaticElement.defaultProps = {
   text: null,
 };
 
+/**
+ * Email element component.
+ * Used to displays plain/formatted email input as part of a form.
```

Comment:
@zaliqarosli I'll redefine the comment here. Also to answer your other comment. I left the module alone without using this component that was made with this PR because of the css is different than a typical form. I didn't feel this PR was targeting that but just the bug of solving correct email verification.

Reply:
> I left the module alone without using this component that was made with this PR because of the css is different than a typical form.

I don't understand why a component is made in this PR but then is not used

Reply:
perhaps this PR includes more code changes than is needed to target the actual bugfix? I'm not quite sure what I'm reviewing here. it looks to me that there are 2 different EmailElement components with the same name, i'm unsure which one is being used, and when it is being used, its to replace a StaticElement (a display) and not an input element 😕

Reply:
Hi @zaliqarosli, I'll delete the component then. As your comment mentions it confuses the review process.